src/deb/control/templates

```diff
@@ -13,7 +13,6 @@ Type: string
 Default: 8004
 Description: The HTTP port you would like content repo to be accessible from
 
-Template: [[artifactId]]/control_port
 Type: string
 Default: 8003
 Description: The TCP port to use for Tomcat shutdown control port
```

Comment:
You should remove the rest of this stanza

Reply:
whoops